client/src/backend/containers/records/export-targets/Form.js

```diff
@@ -0,0 +1,96 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { exportTargetsAPI, requests } from "api";
+import { camelCase } from "lodash";
+import Form from "global/components/form";
+import FormContainer from "global/containers/form";
+
+export default function ExportTargetsContainerForm(props) {
```

Comment:
Would rather you followed general manifold style and went with a class component instead of a functional component.